runtime/gc_ctrl.c

```diff
@@ -335,7 +335,7 @@ void caml_init_gc (void)
   caml_percent_free = norm_pfree (caml_params->init_percent_free);
   caml_gc_log ("Initial stack limit: %"
                ARCH_INTNAT_PRINTF_FORMAT "uk bytes",
-               caml_max_stack_wsize / 1024 * sizeof (value));
```

Comment:
this change is not particularly justified, but I'm fine either way

Reply:
This is in the `caml_init_gc` function which is called *once* per program run.

Reply:
Oh.